prusti-viper/src/encoder/snapshot/encoder.rs

```diff
@@ -1257,8 +1254,9 @@ impl SnapshotEncoder {
                     });
 
                     vir::DomainAxiom {
+                        comment: None,
                         name: format!("{}$len_of_seq", predicate_type.name()),
-                        expr: vir_expr! { forall data: {seq_type} :: { [len_call], [seq_len] } ([len_call] == [seq_len]) },
+                        expr: vir_expr! { forall data: {seq_type} :: { [len_call] } { [seq_len] } :: ([len_call] == [seq_len]) },
```

Comment:
(I understand this change keeps the current behaviour.) Any chance we actually wanted this to be triggered when *both* expressions are in scope?

Reply:
I tried that, but several `prusti-tests` tests related to arrays and slices were failing.